go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	golang.org/x/time v0.3.0 // indirect
 	gomodules.xyz/jsonpatch/v2 v2.3.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/protobuf v1.30.0 // indirect
+	google.golang.org/protobuf v1.33.0 // indirect
```

Comment:
We should override indirect dependencies via `replace`. This can be overridden by the direct dependency.

Reply:
e.g. https://github.com/giantswarm/cluster-api-cleaner-vsphere/pull/99